docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -284,3 +305,83 @@
       environment:
         KUBECONFIG: "/etc/kubernetes/admin.conf"
       when: get_dm_default_registry_key.stdout == ""
+
+    - block:
+        - wait_for:
+            # Waiting task after apply config to avoid failures getting info
+            timeout: 5
+            msg: Waiting after apply DM config
+
+        # - If unlock task is triggered, is highly probable that the config applied is correct.
+        # - If the task fails (by achieving max retries without calling the unlock), the
+        #   playbook will fail but it will collect some information before exiting.
+        - name: Wait until unlock task triggered
+          shell: |
+            source /etc/platform/openrc;
+            system host-show "{{ get_host_name.stdout}}" | awk '$2 ~ /^task/ {print $4}'
+          register: get_show_task_status
+          until: ("Unlocking" in get_show_task_status.stdout)
+          retries: 60
```

Comment:
As the default delay is 5s, here we wait at most 5 min, which should be long enough